### DISCUSSION TITLE
Is there any issue regarding scroller for maths code block ?

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#920] [Q&A]
Hey @fuma-nama, for more details you can visit [this](https://mindect.vercel.app/docs/sl/cost-function-for-logistic-regression/simplified-cost-functionfor-logistic-regression) where I have a large mathematics component, which is causing problems for mobile and tablets.

Was there any issue regarding this in the repository ?

While for other code block their is a horizontal scroller that you have implemented, how can I do this with this component too so that it can be viewed properly on mobile on tablets. 

Is there anything that can be done with this component that I am using 
`rehypeKatex, remarkMath` ? 

Just tell me what can be done ? If changes can be done in fumadocs packages, I am happy to help!!

**Comment by fuma-nama** [comment]
See https://github.com/fuma-nama/fumadocs/discussions/852

**Comment by Gitstar-OC** [comment]
Thanks, just adding this in global.css file 

```css
.katex-html {
   overflow: auto;
  overflow-y: hidden;
}
```
solved my problem..
